### ISSUE TITLE
Extremely minor - Typo in AddOn's configuration interface.  Trivial to fix, and will make a better impression on newbs.

### ISSUE THREAD

**Issue by aprilia1k** [Lej77/tab-count-in-window-title#8]
tl;dr - under the "_Keyboard shortcuts_" configuration,  instructions begin with:
          "Select one of the text inputs **bellow** and then ...".  
   
   The word "bellow" should be changed to "below".

The configuration/management interface for Tab-Count is actually very good, even impressive.  It cleverly employs a variation of a _"drawers"_ paradigm, where the details are hidden until one selects a particular configuration area/type, e.g the above mentioned Keyboard-shortcuts settings.  This keeps things looking neat and uncluttered.

Compared to the average Add-on, this one's interface is clean and well-organized.  The instructions and explanations for settings are straightforward and to-the-point, without sacrificing any clarity.  Seriously it looks good.  This is one more reason the typo stands out - it just doesn't fit this nice UI.  ;-)

*--------------------------------*
Long-winded, silly description:

So, when tapping on the "Keyboard shortcuts" heading to have a look at this feature, the following sentence looked even more silly than it might otherwise have (some stoned newb might actually select an input, then **bellow** something like "_I'm so stoned!_", then proceed to close the window because he forgot what he was doing...  pardon the silly humor please, but this isn't a corporate SMS and we're all friends here, eh?  :-)

"Select one of the text inputs **bellow** and then make a keyboard combination..."

I'll put a tl;dr at the top.   Gotta change "bellow" to "below".

**Comment by Lej77** [comment]
Thank you for all the kind words!

I definitively agree that this typo is silly and should be fixed. So I appreciate that you brought it to my attention. I have released version 5.6 of this extension which includes the fix for this typo among some other small improvements.
